test/options-manager.js

```diff
@@ -190,6 +190,32 @@ test('ignore ignored .gitignore', t => {
 	t.is(ignores.indexOf('!bar/foobar/bar.js'), -1);
 });
 
+test('positive patterns should be translated to negative patterns', t => {
+	const cwd = path.join(__dirname, 'fixtures/gitignore/test');
+	const result = manager.getGitIgnores({cwd});
+
+	t.deepEqual(result, ['!foo.js', '!foo.js/**']);
+});
+
+test('patterns should be translated according to process.cwd()', t => {
+	const previous = process.cwd();
+	const cwd = path.join(__dirname, 'fixtures/gitignore/test');
+	process.chdir(cwd);
+	const result = manager.getGitIgnores({});
+	const expected = ['!foo.js', '!foo.js/**'];
+
+	t.deepEqual(result, expected);
+	process.chdir(previous);
+});
```

Comment:
Because of the test description, I believe the following makes more sense:
```diff
 test('patterns should be translated according to process.cwd()', t => {
        const previous = process.cwd();
-       const cwd = path.join(__dirname, 'fixtures/gitignore/test');
+       const cwd = path.join(__dirname, 'fixtures/gitignore');
        process.chdir(cwd);
        const result = manager.getGitIgnores({});
-       const expected = ['!foo.js', '!foo.js/**'];
+       const expected = ['!test/foo.js', '!test/foo.js/**'];
 
        t.deepEqual(result, expected);
        process.chdir(previous);
 });
```